qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -1397,9 +1430,12 @@ def get_category(self, category):
             If category is not part of the template
         """
         with qdb.sql_connection.TRN:
-            qdb.util.check_table_cols([category], self._table_name(self._id))
-            sql = 'SELECT sample_id, {0} FROM qiita.{1}'.format(
-                category, self._table_name(self._id))
+            if category not in self.categories():
+                raise qdb.exceptions.QiitaDBColumnError(category)
+            sql = """SELECT sample_id, sample_values->>'{0}' as {0}
```

Comment:
pretty similar to `__getitem__`?

Reply:
Yes, but again Sample vs SampleTemplate issue ...

Reply:
ah okay